pingora-core/src/protocols/http/date.rs

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use chrono::NaiveDateTime;
+use chrono::DateTime;
 use http::header::HeaderValue;
 use std::cell::RefCell;
 use std::time::{Duration, SystemTime};
 
 fn to_date_string(epoch_sec: i64) -> String {
-    let dt = NaiveDateTime::from_timestamp_opt(epoch_sec, 0).unwrap();
+    let dt = DateTime::from_timestamp(epoch_sec, 0).unwrap();
```

Comment:
Do we need this change in this PR? We have other changing tracking this issue already

Reply:
this pr causes us to run a newer version of clippy (because the old pipeline was deprecated/using an old version) so it will fail the build if this change is not in it.

Reply:
The `chrono` deprecation fix was merged, so we should be able to remove this commit from the PR.

Reply:
ok will do that now